relay-server/src/actors/store.rs

```diff
@@ -1100,13 +1104,26 @@ impl Message for KafkaMessage<'_> {
             Self::Attachment(message) => message.event_id.0,
             Self::AttachmentChunk(message) => message.event_id.0,
             Self::UserReport(message) => message.event_id.0,
-            Self::Session(_message) => Uuid::nil(), // Explicit random partitioning for sessions
-            Self::Metric { .. } => Uuid::nil(),     // TODO(ja): Determine a partitioning key
-            Self::Profile(_message) => Uuid::nil(),
             Self::ReplayEvent(message) => message.replay_id.0,
-            Self::ReplayRecordingNotChunked(_message) => Uuid::nil(), // Ensure random partitioning.
-            Self::CheckIn(_message) => Uuid::nil(),
-            Self::Span(_) => Uuid::nil(), // random partitioning
+
+            // Random partitioning
+            Self::Session
+            | Self::Session
+            | Self::Metric
+            | Self::Profile
+            | Self::ReplayRecordingNotChunked
+            | Self::Span => Uuid::nil(),
```

Comment:
Organized this a bit better I think, clearer separation between message types that are being randomly partitioned and not.